themes/frontend-theme-valencia/src/css/_custom.scss

```diff
@@ -0,0 +1,492 @@
+/* inject:imports */
+
+
+/* endinject */
+
+/* ---------- Fonts ---------- */
+@font-face {
+    font-family: 'roboto';
+    src: url('../fonts/Roboto-Light-webfont.woff') format('woff');
+    font-weight: 300;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'roboto';
+    src: url('../fonts/Roboto-Regular-webfont.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'roboto';
+    src: url('../fonts/Roboto-Medium-webfont.woff') format('woff');
+    font-weight: 500;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'roboto';
+    src: url('../fonts/Roboto-Bold-webfont.woff') format('woff');
+    font-weight: 700;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'roboto_slab';
+    src: url('../fonts/RobotoSlab-Regular-webfont.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'roboto_slab';
+    src: url('../fonts/RobotoSlab-Bold-webfont.woff') format('woff');
+    font-weight: 700;
+    font-style: normal;
+}
+
+/* ---------- Sass Variables ---------- */
+
+$color-bg: #E5E5E5;
+$color-font-header: #3C4858;
+$color-font-body: #555;
+$color-theme-main: #7DBE0D;
+$color-theme-secondary: #0AACCC;
+$color-theme-tertiary: #E8990C;
+$font-sans: roboto, Helvetica, Arial, sans-serif;
+$font-serif: roboto_slab, "Palatino Linotype", Georgia, serif;
+
+/* ---------- Generic Styles ---------- */
+
+body {
+    background-color: $color-bg;
+	color: $color-font-body;
```

Comment:
I think you are mixing spaces and tabs since this line is not aligned as the previous line.